index.html

```diff
@@ -11,7 +11,8 @@
     <script src="https://kit.fontawesome.com/2c36e9b7b1.js" crossorigin="anonymous"></script>
     <meta property="og:title" content="Tech Biscuits">
     <meta property="og:type" content="website">
-    <meta property="og:description" content="TechBiscuits is a website where you can find cool and creative DIY projects made by two friends.">
+    <meta property="og:description" content="TechBiscuits is a small software company specialising in privacy-focused mobile and web applications.">
```

Comment:
bruva you know what happened when we did this last time

Reply:
I don't